docs/data/date-pickers/time-range-picker/time-range-picker.md

```diff
@@ -0,0 +1,18 @@
+---
+product: date-pickers
+title: React Time Range Picker component
+components: TimeRangePicker, DesktopTimeRangePicker, MobileTimeRangePicker, StaticTimeRangePicker
```

Comment:
It creates 404 on the footer https://deploy-preview-4603--material-ui-x.netlify.app/x/react-date-pickers/time-range-picker/

```suggestion
```